### ISSUE TITLE
Custom fields only supported in returned values, but not search criteria?

### ISSUE THREAD

**Issue by OsoRojo** [marc0l92/obsidian-jira-issue#26]
Is it true that the custom field $ syntax is only usable in the returned fields section?  Most of my critical fields are custom, and I need to be able to actually query on them, but thus far I get a slew of errors when trying.  Namely that $ is a reserved word in JQL (if I sniff the dev console and try and reuse the API call as it appears)

**Comment by OsoRojo** [comment]
Side note, this could be user error.  I did get SOMETHING to work using the query builder and a custom JQL query.  But what the query builder used and what I deduced from my install UI and the documentation are two different things:

What I tried to use:
`$Team = 'Remotely Agile'

What worked (at least somewhat)
`"Team[Team]" = 57`

**Comment by marc0l92** [comment]
The syntax of the JQL query is the same as the one you use on Jira web UI when you create your dashboards. This plugin takes the query you input and it send it to the server using the APIS.

The $ symbol is something I made up to be used in the returned fields. I decided to introduce this symbol because Jira allows to introduce custom fields that have the same name as the standard one. In order to distinguish between the two I introduced this concept of $ prefix.

To summarize: in the JQL use the official syntax of Jira and don't put any `$` symbol
